src/consistency-testing/chaostest/chaostest/redpanda_cluster.py

```diff
@@ -237,7 +237,7 @@ async def is_ok(self):
             host = endpoint["host"]
             port = endpoint["httpport"]
             address = f"{host}:{port}"
-            kv = KVNode(address, address)
+            kv = KVNode(0, address, address)
```

Comment:
was this intended to be endpoint["idx"] (and the couple other cases of using `0` above)?

Reply:
It works both ways, idx is used externally by the consumers of KVNode when they log latency, in this place we use KVNode directly and then abandon it so there are no consumers and it doesn't matter what's the actual value of idx here.

But I think it's better to keep code uniform so I'll change it not to confuse a reader

Reply:
sounds good. i approved if you want to make that change or not.

Reply:
thanks, I've fixed it!